<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideMenu</class>
 <widget class="QWidget" name="SideMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="playBtn">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Play (Space)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unit (W/S)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showAll">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Show all (A)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="units">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>QuantX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="quantX">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>QuantY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="quantY">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
